modules/logger/parser/store.go

```diff
@@ -0,0 +1,110 @@
+// Package logger/parser contains methods to parse and restructure log output from go testing and terratest
+package parser
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/gruntwork-io/terratest/modules/files"
+	junitformatter "github.com/jstemmer/go-junit-report/formatter"
+	junitparser "github.com/jstemmer/go-junit-report/parser"
+	"github.com/sirupsen/logrus"
+)
+
+type LogWriter struct {
+	// Represents an open channel to a test writer for given test indexed by name
```

Comment:
Curious why you need channels for this?

Reply:
I was thinking about the writer as a task, that would need to open the file and write to it and then I landed on channels and goroutines where the task would open the file and wait for input.
... and then this comment made me realize that I can just open the file handle and store that here and write to that.... 

![picarddoublefacepalm-1](https://user-images.githubusercontent.com/430092/47269685-5103cb00-d516-11e8-810b-a43e73fc47d1.jpg)

Reply:
😄

Reply:
😄

Reply:
😄

Reply:
fixed